content/academy/apify_platform/deploying/inputs_outputs.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Inputs & outputs
+description: Learn to accept input into your actor, do something with it, then return output. Actors can be written in any language, so this concept is language agnostic.
+menuWeight: 1
+paths:
+    - apify-platform/deploying/inputs-outputs
+---
+
+# [](#inputs-outputs) Inputs & outputs
+
+Most of the time when you're creating a project, you are expecting some sort of input off of which your software will run off of. Often times as well, you want to provide some sort of output once your software has completed running. With Apify, it is extremely easy to take in inputs and deliver outputs.
+
+An important thing to understand regarding inputs and outputs is that they are read/written differently depending on where the actor is running:
+
+- If your actor is running locally, the inputs/outputs are usually provided in the filesystem, and environment variables are injected either by you, the developer, and by the Apify CLI by running the project with the `apify run` command.
```

Comment:
Just an idea. What about moving the generic and installation part of the Apify CLI (in deploying) to glossary/tools? That way, we could link to it, for example from here. And the deploying lesson would reference it for installation and only continue with the deploying part.

I think this could be useful, because we will be talking about Apify CLI in many places, so we could have one "crossroads" for the user. We could even have bi-directional links - e.g. link from deploying to Apify CLI concept, but also link from the concept to the deploying section.

Reply:
really good idea